### PR TITLE
[CR] Adds wearable and edible candy bracelets

### PR DESCRIPTION
**Pull request by Doitallornot123**
#### Summary
Content "Adds wearable candy bracelets"

#### Purpose of change
Content, no matter how insignificant, is always nice. This is my first attempt at a PR and I look forward to learning and contributing more.

#### Describe the solution
Adds wearable and edible candy bracelets to the game in the "candy_chocolate" item group

#### Describe alternatives you've considered
Adding candy necklaces too, or adding none at all.

#### Testing
Started the game, created a new character, loaded into the game with no errors, spawned the item via the debug menu successfully, item functioned as expected.

#### Additional context
i understand and apologize if the four commits seem odd, this was also me learning how github works...

### CODE REVIEW

**Comment by NetSysFire** [comment]
You should keep your master branch clean as described in the contributing guidelines. Otherwise it will cause you some trouble when you try to file more PRs or rebase.

**Comment by Doitallornot123** [comment]
> 

I intend to clean it up when this PR gets resolved. I'm very new to this and am learning as I go. I do appreciate the advice

File: data/json/items/comestibles/junkfood.json [hunk 1]
```diff
@@ -462,6 +462,16 @@
     "charges": 5,
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "candy_bracelet",
+    "name": { "str_sp": "candy bracelet" },
+    "looks_like": "copper_bracelet",
+    "color": "yellow",
+    "description": "A candy bracelet made by stringing candies onto a thread. In theory, you could both wear and eat the bracelet.",
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:471]:
```suggestion
    "description": "A candy bracelet made by stringing candies onto a thread.  In theory, you could both wear and eat the bracelet.",
```
Double space after a full stop

Comment (Doitallornot123) [reply data/json/items/comestibles/junkfood.json]: done, i appreciate the advice

**Comment by Doitallornot123** [comment]
> You should keep your master branch clean as described in the contributing guidelines. Otherwise it will cause you some trouble when you try to file more PRs or rebase.

**Comment by Doitallornot123** [comment]
it seems like the item density is higher than is allowed. but I don't understand how that's possible as I haven't changed the items volume or mass from the basic candy4 item. Any advice is appreciated.

**Pull request merged**